exercises/resistor-color-duo/.meta/hints.md

```diff
@@ -8,3 +8,32 @@ value :: (Color, Color) -> Int
 
 that only accepts two colors and produces the resistor's numeric value
 component of its resistance.
+
+**Bonus - Property Testing:**
+
+This exercise is a good opportunity to write some [property tests](https://en.wikipedia.org/wiki/Property_testing). If you haven't written any before here are some resources:
+
+1. [School of Haskell](https://www.schoolofhaskell.com/user/pbv/an-introduction-to-quickcheck-testing).
+2. [Real world Haskell](http://book.realworldhaskell.org/read/testing-and-quality-assurance.html)
+
+We have provided one sample property test in the [test file](test/Tests.hs). We use the [QuickCheck](https://hackage.haskell.org/package/QuickCheck) library, but there are [others](http://hackage.haskell.org/package/hedgehog) as well, feel free to play around.
```

Comment:
- sample -> example
- I don't think linking to test/Tests.hs will work on the website.

```suggestion
We have provided one example property test in the test file. We use the [QuickCheck](https://hackage.haskell.org/package/QuickCheck) library, but there are [others](http://hackage.haskell.org/package/hedgehog) as well, feel free to play around.
```

Alternatively, since Hedgehog isn't beginner-friendly, skip mentioning it altogether. In which case mentioning that we use QuickCheck may be considered redundant. In which case this whole sentence could technically be omitted.

Reply:
I take this back. Mentionining the example and that we use QuickCheck is indeed useful.